source/crud/tests/README.md

```diff
@@ -737,3 +737,15 @@ that `firstEvent.operationId` is equal to `secondEvent.operationId`. Assert both
 To force completion of the `w:0` writes, execute `coll.countDocuments` and expect the returned count is
 `maxMessageSizeBytes / maxBsonObjectSize + 1`. This is intended to avoid incomplete writes interfering with other tests
 that may use this collection.
+
+### 16. Generated document identifiers are the first field in their document
```

Comment:
Looking at this test, aside from asserting that `_id` is the first field in the document, I don't see a reason why this should be a prose test and can't be a unified test. Keep in mind that prose test cause significantly more downstream implementation effort than unified spec tests.

Reply:
The explicit purpose of the test is to assert that `_id` is the first field in the document. Unless we add the capability to check mapping order to the unified spec runner, a prose test is the only way to do this.

Reply:
I agree with Noah here, as a driver that could benefit from a key order solution in the unified spec runner, I don't think adding support for something like that is reasonable to finish within spec-fest, but paying the cost of prose for generally improving write performance wherever this advice is not followed is worth paying, in my opinion.

Reply:
I had missed the part where this was specifically testing that `_id` is the first field in the document, god confused with all the `_id` pull requests. I agree that the prose test makes sense, especially since this assertion is not possible in the unified test format.